### PR TITLE
LPD-19789 - Add default view field to Data Set model

### PR DESCRIPTION
**Pull request by juanjofgliferay**
## Story / Task
[LPD-6630](https://liferay.atlassian.net/browse/LPD-6630) / [LPD-19789](https://liferay.atlassian.net/browse/LPD-19789)

## Goal
Define a DB field to store the default view of a Data Set in order to be used when rendering the fragment

### CODE REVIEW

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/portlet/FDSViewsPortlet.java [hunk 1]
```diff
@@ -689,6 +689,32 @@ private ObjectDefinition _createFDSViewObjectDefinition(
 			fdsViewObjectDefinition.getObjectDefinitionId(),
 			labelObjectField.getObjectFieldId());
 
+		if (FeatureFlagManagerUtil.isEnabled("LPD-10735")) {
+			ObjectField defaultViewObjectField =
```
Comment (juanjofgliferay) [inline modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/portlet/FDSViewsPortlet.java:693]: Don't know if it is worth all this boilerplate to add a field under a FF. 
An option could be to customize the [_addLocalizedCustomObjectField](https://github.com/liferay-frontend/liferay-portal/blob/5f5adf9cd182d06f693d01128a59600f259f47f9/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/portlet/FDSViewsPortlet.java#L150) and allow creating localized/non-localized custom object fields.

Comment (dsanz) [reply modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/portlet/FDSViewsPortlet.java:693]: Forgot to say, anyways, I'm ok with the code even if it's not optimal. In the end we'll end up removing it soon when migrating the data model

Comment (juanjofgliferay) [reply modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/portlet/FDSViewsPortlet.java:693]: If we are not going to break anything, my bet would be adding the field directly, without FF.

**Comment by juanjofgliferay** [comment]
ci:test:relevant

**Comment by juanjofgliferay** [comment]
ci:test:relevant

**Pull request closed without merging**